src/Compilers/CSharp/Portable/Parser/SyntaxParser.cs

```diff
@@ -287,7 +287,7 @@ protected SyntaxToken CurrentToken
         {
             get
             {
-                return _currentToken ?? (_currentToken = this.FetchCurrentToken());
+                return _currentToken ??= this.FetchCurrentToken();
```

Comment:
As much as I'm a fan of using newer features, probably not worth changing existing code.